druid/src/contexts.rs

```diff
@@ -495,6 +495,14 @@ impl EventCtx<'_, '_> {
             );
         }
     }
+
+    /// Request an update cycle.
+    ///
+    /// After this, `update` will be called on the widget in the next update cycle, even
+    /// if there's not a data change.
```

Comment:
I think it's worth highlighting that this method is only exposed for special cases, such as when a container widget is synthesizing data for its children, and that in general it should be unnecessary/avoided.